src/Microsoft.CodeAnalysis.Analyzers/UnitTests/MetaAnalyzers/PreferIsKindAnalyzerTests.cs

```diff
@@ -0,0 +1,423 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Testing;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
+    Microsoft.CodeAnalysis.Analyzers.MetaAnalyzers.PreferIsKindAnalyzer,
+    Microsoft.CodeAnalysis.CSharp.Analyzers.MetaAnalyzers.Fixers.CSharpPreferIsKindFix>;
+using VerifyVB = Test.Utilities.VisualBasicCodeFixVerifier<
+    Microsoft.CodeAnalysis.Analyzers.MetaAnalyzers.PreferIsKindAnalyzer,
+    Microsoft.CodeAnalysis.VisualBasic.Analyzers.MetaAnalyzers.CodeFixes.BasicPreferIsKindFix>;
+
+namespace Microsoft.CodeAnalysis.Analyzers.UnitTests.MetaAnalyzers
+{
+    public class PreferIsKindAnalyzerTests
+    {
+        [Theory]
+        [InlineData("==")]
+        [InlineData("!=")]
```

Comment:
Consider testing `is` and `is not` operators as well. Probably with C# 9.0 patterns (`and`, `or`).

Reply:
i actually think if you're doing `and` then this form is likely nicer to read :)

Reply:
I'm excluding these for now since this analyzer package doesn't reference a new enough version of Microsoft.CodeAnalysis.Common to include `INegatedPatternOperation`.